scripts/e2e-tests.sh

```diff
@@ -193,11 +193,8 @@ function create_test_cluster() {
   mkdir -p $HOME/.ssh
   touch $HOME/.ssh/google_compute_engine.pub
   touch $HOME/.ssh/google_compute_engine
-  # Clear user and cluster variables, so they'll be set to the test cluster.
-  # DOCKER_REPO_OVERRIDE is not touched because when running locally it must
+  # KO_DOCKER_REPO is not touched because when running locally it must
```

Comment:
Please remove this comment as well. You're now dealing with KO_DOCKER_REPO below, not here.

Reply:
Done